docs/downstream-integrations/api/token-gating.md

```diff
@@ -3,10 +3,13 @@ id: token-gating
 title: Token Gating
 sidebar_position: 4
 ---
-
-The `/access-control` endpoint allows client applications to interact with Collab.Land's token gating engine to verify a wallet address or crypto account with a list of rules based on the token ownership.
+## Overview
+The `/access-control` [endpoint](link needed) allows client applications to interact with Collab.Land's token gating engine to verify a wallet address or crypto account with a list of rules based on the blockchain token ownership.
```

Comment:
link needed here

Reply:
@raymondfeng what linking practice should we follow?

Reply:
@uniyalabhishek he suggested using [relative links](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#relative-links) I believe 

https://github.com/abridged/collabland-dev/pull/26#discussion_r1028778459